util/src/main/java/io/camunda/zeebe/util/sched/ActorThread.java

```diff
@@ -77,7 +78,12 @@ private void doWork() {
 
     if (currentTask != null) {
       try {
-        executeCurrentTask();
+        final var actorName = currentTask.actor.getName();
+        actorMetrics.countExecution(actorName);
```

Comment:
:upside_down_face: I would increment after we've executed the task, since I understand the metric to mean it _was_ executed. :man_shrugging: